errors/main.go

```diff
@@ -5,11 +5,11 @@ import "errors"
 type RailwayError error
 
 var (
-	UserConfigNotFound      RailwayError = errors.New("Not logged in. Please run railway login.")
+	UserConfigNotFound      RailwayError = errors.New("Not logged in. Run railway login.")
 	ProjectConfigNotFound   RailwayError = errors.New("Not connected to a project. Run railway init to get started.")
-	ProjectNotFound         RailwayError = errors.New("Project not found.")
-	ProblemFetchingProjects RailwayError = errors.New("There was a problem fetching your projects")
-	ProjectCreateFailed     RailwayError = errors.New("There was a problem creating the project")
-	ProductionTokenNotSet   RailwayError = errors.New("RAILWAY_TOKEN environment variable not set")
+	ProjectNotFound         RailwayError = errors.New("Project not found.\nTry railway init to get plugged into a new or existing project.")
+	ProblemFetchingProjects RailwayError = errors.New("There was a problem fetching your projects.\nOne of our trains probably derailed!")
+	ProjectCreateFailed     RailwayError = errors.New("There was a problem creating the project\nOne of our trains probably derailed!")
+	ProductionTokenNotSet   RailwayError = errors.New("RAILWAY_TOKEN environment variable not set.\nRun railway open project and head under `tokens` section. You can generate tokens to access Railway environment variables. Set that token into your terminal by using `export RAILWAY_TOKEN=<insert token>` and you're all aboard!")
```

Comment:
Should just say to set the environment variable for the executable, since there are many ways to set it `RAILWAY_TOKEN=.... railway run`, or via cmd source, etc